### PR TITLE
Add drop down/up component

### PR DESCRIPTION
**Pull request by kaytcat**

### CODE REVIEW

File: components/DropDown/index.jsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  }
+
+  state = {
+    isMounted: true,
+    isOpen: false,
+    selected: this.getSelection()
+  }
+
+  componentWillMount () {
+    document.addEventListener('click', this.handleDocumentClick, false);
```
Comment (feychou) [inline components/DropDown/index.jsx]: `onClickDocument` would be the standard way to name callback in handlers we agreed on.

File: components/DropDown/index.jsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+import styles from './index.scss';
+
+import {
+  clearfix
+} from '../index.scss';
+
+export default class Dropdown extends Component {
+  static propTypes = {
+    items: PropTypes.array.isRequired,
```
Comment (feychou) [inline components/DropDown/index.jsx]: We could specify it is an array of objects. And possibly specify the shape of the objects themselves ^^

**Comment by feychou** [comment]
Can we save the data images somewhere?

File: components/DropDown/index.jsx (hunk 2 above)
Comment (kaytcat) [inline components/DropDown/index.jsx]: Yes!! good idea :D

**Pull request merged**